tests/api_connexion/endpoints/test_dag_endpoint.py

```diff
@@ -255,12 +256,123 @@ def test_should_response_200_default_limit(self):
+                    "owners": [],
+                    "root_dag_id": None,
+                    "schedule_interval": {
+                        "__type": "CronExpression",
+                        "value": "1 1 * * *",
+                    },
+                    "tags": [],
+                },
+            ),
+            (
+                "patch time delta",
+                {
+                    "is_paused": False,
+                    "schedule_interval": {
+                        "__type": "TimeDelta",
+                        "days": 1,
+                        "seconds": 1,
+                        "microseconds": 1,
+                    },
+                },
+                {
+                    "dag_id": "TEST_DAG_1",
+                    "description": None,
+                    "fileloc": "/tmp/dag_1.py",
+                    "is_paused": False,
+                    "is_subdag": False,
+                    "owners": [],
+                    "root_dag_id": None,
+                    "schedule_interval": {
+                        "__type": "TimeDelta",
+                        "days": 1,
+                        "microseconds": 1,
+                        "seconds": 1,
+                    },
+                    "tags": [],
+                },
+            ),
+            (
+                "patch relative delta",
+                {
+                    "is_paused": False,
+                    "schedule_interval": {
+                        "__type": "RelativeDelta",
+                        "year": 1,
+                        "months": 2,
+                        "days": 1,
+                        "seconds": 1,
+                        "microseconds": 1,
+                    },
+                },
+                {
+                    "dag_id": "TEST_DAG_1",
+                    "description": None,
+                    "fileloc": "/tmp/dag_1.py",
+                    "is_paused": False,
+                    "is_subdag": False,
+                    "owners": [],
+                    "root_dag_id": None,
+                    "schedule_interval": {
+                        "__type": "RelativeDelta",
+                        "day": None,
+                        "days": 1,
+                        "hour": None,
+                        "hours": 0,
+                        "leapdays": 0,
+                        "microsecond": None,
+                        "microseconds": 1,
+                        "minute": None,
+                        "minutes": 0,
+                        "month": None,
+                        "months": 2,
+                        "second": None,
+                        "seconds": 1,
+                        "year": 1,
+                        "years": 0,
+                    },
+                    "tags": [],
+                },
+            ),
+        ]
+    )
+    @provide_session
+    def test_should_response_200(self, name, patch_body, expected_reponse, session):
+        del name
```

Comment:
Looks like the pattern for passing in an unused name variable, according to the docs, is to rename `name` as `_`. Then you don't have to delete the value.

Reply:
Neat :smile:! Fixed it.